td.vue/package.json

```diff
@@ -58,6 +60,8 @@
     "browserstack-cypress-cli": "^1.8.1",
     "chromedriver": "^90.0.0",
     "codecov": "^3.8.2",
+    "electron": "^13.0.0",
+    "electron-devtools-installer": "^3.1.0",
```

Comment:
Could we use the latest version of electron-devtools-installer which is 3.2.0 ?

Reply:
Done!